ui/components/component-library/button-primary/README.mdx

```diff
@@ -12,33 +12,30 @@ The `ButtonPrimary` is an extension of `ButtonBase` to support primary styles.
 
 ## Props
 
-The `ButtonPrimary` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) and [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
+The `ButtonPrimary` accepts all props below as well as all [Box](/docs/components-componentlibrary-box--docs#props) and [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
```

Comment:
As discussed we can remove this line altogether moving forward as all props are provided including the style utility props from the Box component in the prop table